src/funcs.c

```diff
@@ -1397,6 +1432,14 @@ void game_properties(void)
 	}
 
 	// Cleanup the memory allocations
+#if defined(__amigaos4__)
+	FreeVec(slave);
```

Comment:
You try to free memory without checking if this is already reserved. That might lead to crashes.

Reply:
AFAIK latest SDK (OS4) FreeVec() allows NULL values/pointers (it just doesn't do anything).